runtime/kusama/src/xcm_config.rs

```diff
@@ -169,14 +165,20 @@ pub type LocalOriginToLocation = (
 );
 impl pallet_xcm::Config for Runtime {
 	type Event = Event;
-	type SendXcmOrigin = xcm_builder::EnsureXcmOrigin<Origin, LocalOriginToLocation>;
+	// We don't allow any messages to be sent via the transaction yet. This is basically safe to
+	// enable, (safe the possibility of someone spamming the parachain if they're willing to pay
+	// the DOT to send from the Relay-chain). But it's useless until we bring in XCM v3 which will
+	// make `DescendOrigin` a bit more useful.
+	type SendXcmOrigin = xcm_builder::EnsureXcmOrigin<Origin, ()>;
 	type XcmRouter = XcmRouter;
-	// Anyone can execute XCM messages locally...
+	// Anyone can execute XCM messages locally.
 	type ExecuteXcmOrigin = xcm_builder::EnsureXcmOrigin<Origin, LocalOriginToLocation>;
-	// ...but they must match our filter, which rejects all.
-	type XcmExecuteFilter = Nothing;
+	type XcmExecuteFilter = Everything;
```

Comment:
Setting this to `Everything` means that anyone can call `execute` on the XCM pallet and provide an arbitrary XCM as an extrinsic to be executed. If we're 100% confident that XCM is good to go at its current form, then there shouldn't be any issues setting it to `Everything`, but just making sure we're on the same page.

Reply:
Ah, I see, this is for Kusama, definitely not a huge problem then.